docs/roadmap.md

```diff
@@ -60,8 +60,8 @@ The [GitOps Engine](https://github.com/argoproj/gitops-engine) is a library that
 A lot of Argo CD features are still not available in GitOps engine. The following features have to be contributed to the GitOps Engine:
 
 * an ability to customize resources health assessment and existing CRD health [assessment functions](https://github.com/argoproj/argo-cd/tree/master/resource_customizations).
-* resource diffing [customization](https://argoproj.github.io/argo-cd/user-guide/diffing/).
-* config management [tools](https://argoproj.github.io/argo-cd/user-guide/application_sources/) integration.
+* resource diffing [customization](../user-guide/diffing/).
+* config management [tools](../user-guide/application_sources/) integration.
```

Comment:
I think `../user-guide` can just be `/user-guide/` here as well

Reply:
We have to use relative URL since docs are served under sub-path (`https://argo-cd.readthedocs.io/en/stable/` instead of `https://argo-cd.readthedocs.io/`)